src/app.js

```diff
@@ -336,6 +336,7 @@ function run () {
   // ---------------- ContextView -----------------------
   this._components.contextView = new ContextView({
     contextualListener: this._components.contextualListener,
+    editor: editor,
```

Comment:
don't see any use of editor in `contextView` ... do you need this ref?

Reply:
Removed!